### PR TITLE
Removed Activity Log solution from Terraform RI

### LINKED ISSUE #553: Remove Activity Log solution from Terraform RI
### Description

As per ADO USER STORY 25142
1. Remove Activity Log solution from Terraform RI 
2. Update and Terraform documentation referencing this

### PR DESCRIPTION
**Pull request by lachaves**
## Overview/Summary

As per ADO USER STORY 25142

1. Remove Activity Log solution from Terraform RI
2. Update and Terraform documentation referencing this

## This PR fixes/adds/changes/removes

1. Fixes #553 

### Breaking Changes

1.  Need validation from team

## Testing Evidence

Please provide any testing evidence to show that your Pull Request works/fixes as described and planned (include screenshots, if appropriate).

ALZ tf deployment

examples-deploy-management-resources

## As part of this Pull Request I have

- [x ] Checked for duplicate [Pull Requests](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/pulls)
- [ x] Associated it with relevant [issues](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues), for tracking and closure.
- [x ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/tree/main)
- [x ] Performed testing and provided evidence.
- [ x] Updated relevant and associated documentation.
- [X ] Updated the ["RELEASE"](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues) page with details needed for when this code change is released.

### CODE REVIEW

**Review by krowlandson — CHANGES_REQUESTED** [review]
Looks good @lachaves... just one thing missing that I can see.

Please can you update `docs/wiki/media/examples-deploy-management-custom-resources.png` as per [`docs/wiki/media/examples-deploy-management-resources.png`](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/pull/554/files#diff-64cf8a91cd99250366a6a665c2921f15bc7c022d8006b238c53c2f160cefec1a)?

I believe these are actually the same image, but referenced by different names in case we decide to customise the list of solutions in the example at some point. Perhaps this might be a good time to do this actually? 👍🏻

**Comment by lachaves** [comment]
@krowlandson, sure, so the idea is to remove some of the solutions and have a different image for custom management solutions, right? Anything else for the new image?

**Comment by lachaves** [comment]
@krowlandson, Ill fix the Code Review/Lint errors so it can finish the checks

File: docs/wiki/[Examples]-Deploy-Management-Resources-With-Custom-Settings.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ In this example, we take the base [Deploy Management resources][wiki_deploy_mana
   - Disable Azure Defender for Azure Kubernetes Service (AKS)
   - Set a different location for Management resources (controlled through an input variable on the root module)
   - Add custom resource tags for Management resources (controlled through an input variable on the root module)
+  - Set the local `configure_management_resources` input variable in the  `settings.management.tf` file for a custom deployment of the management resources solutions. If set to false the solution will not be deployed.
```
Comment (krowlandson) [inline docs/wiki/[Examples]-Deploy-Management-Resources-With-Custom-Settings.md:18]: Just an FYI... I am updating this to a more succinct statement

File: docs/wiki/[Examples]-Deploy-Management-Resources-With-Custom-Settings.md [hunk 2]
```diff
@@ -101,7 +102,7 @@ variable "security_alerts_email_address" {
 
 variable "management_resources_location" {
   type    = string
-  default = "uksouth"
+  default = "eastus"
```
Comment (krowlandson) [inline docs/wiki/[Examples]-Deploy-Management-Resources-With-Custom-Settings.md:105]: Just an FYI, I am reverting this to `uksouth` as `eastus` is the global module default for this and the point of the custom demo is to set a different location.

**Review by krowlandson — CHANGES_REQUESTED** [review]
Thank you @lachaves 

I'm addressing my comments on this review pass, but please can you ensure the images for the wiki reflect the correct locations for default (`eastus`) and custom (`uksouth`)?

**Review by krowlandson — APPROVED** [review]
LGTM 🚀

**Pull request merged**